src/popup/router/components/Loader.vue

```diff
@@ -3,91 +3,51 @@
     class="loader"
     data-cy="loader"
   >
-    <div class="center">
-      <span v-if="content !== ''">{{ content }}</span>
-      <br v-if="content !== ''">
-      <ae-loader v-if="size === 'small'" />
+    <div class="container">
+      <AnimatedSpinner class="spinner" />
     </div>
-    <transition v-if="size === 'big'">
-      <span
-        v-if="type === 'transparent'"
-        class="main-loader main-loader-transparent"
-      >
-        <ae-loader />
-      </span>
-      <Welcome
-        v-else
-        class="main-loader main-loader-solid"
-      />
-    </transition>
   </div>
 </template>
 
 <script>
-import Welcome from './Welcome';
+import AnimatedSpinner from '../../../icons/animated-spinner.svg?skip-optimize';
 
 export default {
-  components: {
-    Welcome,
-  },
-  props: {
-    content: { type: String, default: '' },
-    size: { type: String, default: 'big' },
-    type: { type: String, default: 'transparent' },
-  },
+  components: { AnimatedSpinner },
 };
 </script>
 
 <style lang="scss" scoped>
 @use '../../../styles/variables';
+@use '../../../styles/mixins';
 
 .loader {
-  .center {
-    position: relative;
-    z-index: 5;
-    word-break: break-word;
-  }
-
-  .ae-loader {
-    border: 0.2em solid variables.$color-blue;
-    border-left-color: transparent;
-    border-right-color: transparent;
-  }
-
-  .main-loader {
-    position: fixed;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: 9;
```

Comment:
Maybe let's get rid of z-indexes bigger than 1 (in Header for example) and set this to 2. We also have corresponding issue #1171